.canon.yaml

```diff
@@ -8,3 +8,13 @@ pi:
     user: testbot
     group: testbot
     persistent: true
+
+pi-32:
+    arch: arm
+    image_arm: ghcr.io/viam-modules/raspberry-pi:arm32
```

Comment:
No need fore a new profile, just add this one line to the main profile. People can then switch with just `-arch arm` since everything else is the same.